queries/faust/highlights.scm

```diff
@@ -0,0 +1,239 @@
+; Identifiers
+(process) @variable.builtin
+
+(parameters
+  (identifier) @variable.parameter)
+
+(identifier) @variable
+
+; Literals
+(file_import
+  filename: (string)) @string.special.path
+
+(soundfile
+  filename: (string)) @string.special.path
+
+(component
+  filename: (string)) @string.special.path
+
+(library
+  filename: (string)) @string.special.path
```

Comment:
```suggestion
(_
  filename: (string)) @string.special.path
```

Reply:
❤️ exactly what I was looking for, thanks!

Reply:
That reminds me - can I somehow use list brackets `[...]` to simplify
```
(file_import filename: (string)) @string.special.path
(soundfile filename: (string)) @string.special.path
(component filename: (string)) @string.special.path
(library filename: (string)) @string.special.path
```
 - does it work with parent rules?

Reply:
nah, sorry - it'd be difficult to know which rule you're targeting then

Reply:
Ok that's what I figured, thanks